### ISSUE TITLE
Accessibility of nav_item.html and topic.html

### ISSUE THREAD

**Issue by ibrahimjaved12** [mitodl/ocw-hugo-themes#1220]
Open up: https://ocw.mit.edu/courses/15-s21-nuts-and-bolts-of-business-plans-january-iap-2014/

## Accessibility of course-v2/layouts/partials/nav_item.html

Code:
```
    <a class="course-nav-section-toggle"
      href="#nav-container_{{ $device }}_{{ .menuItem.Identifier }}"
      data-toggle="collapse"
      aria-controls="nav-container_{{ $device }}_{{ .menuItem.Identifier }}"
      aria-expanded=""
      data-uuid="{{ .menuItem.Identifier }}">
      <i class="material-icons md-18"></i>
    </a>
```
This does not have `aria-label`. Also, `aria-expanded` should be set to either `true` or `false`.
## Screenshot
<img width="225" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/44dec57c-cd46-4ee4-8765-30302c90e5e1">
The arrow here is not accessiblility-friendly

## Accessibility of course-v2/layouts/partials/topic.html:
If we press the toggle button for collapse/show, each time it gives a different label depending on the kind of page we are on.
Eg., here, https://ocw.mit.edu/courses/15-s21-nuts-and-bolts-of-business-plans-january-iap-2014/, it gives the label of course name, like `15-s21-nuts-and-bolts-of-business-plans-january-iap-2014`.
But if we are on the instructor insights page, it gives the label of` instructor insights` from the right menu. And if we are on download page, it gives the label of `download`, even though the nature of pages has nothing to do with the nature of the toggle button.

## Screenshot

<img width="133" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/37de0e34-9c8f-44ff-8920-9ec7937774cc">

The arrow here is not accessibility-friendly
